### PR TITLE
WIP: Proxy to alertmanager, use envsubst for service IPs

### PR DESCRIPTION
**Pull request by jsravn**
part of https://github.com/lightbend/es-backend/issues/102

### CODE REVIEW

**Comment by jsravn** [comment]
I'm not sure why, but this change causes nginx to start redirecting to port 8080. I'm sort of baffled :thinking: .

File: enterprise-suite/templates/es-grafana.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ metadata:
   labels:
     app: grafana
     component: "server"
-  name: grafana-server
+  name: grafana
```
Comment (marcoderama) [inline enterprise-suite/templates/es-grafana.yaml]: Do we need to let people know about this change (and the prometheus one)?

File: enterprise-suite/templates/es-grafana.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ spec:
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: grafana-server
+  name: grafana
```
Comment (marcoderama) [inline enterprise-suite/templates/es-grafana.yaml]: Does line 106 need to change too?  
    `name: grafana-server`
What about the comment at line 11?
    `# 2. add prometheus data source to grafana-server.`

File: enterprise-suite/templates/es-console.yaml [hunk 3]
```diff
@@ -14,6 +14,25 @@ spec:
...
+        image: alpine
+        resources:
+          requests:
+            cpu: {{ default .Values.defaultCPURequest }}
+            memory: {{ default .Values.defaultMemoryRequest }}
+        command:
+        - /bin/sh
+        - -c
+        args:
+        - apk add gettext libintl && cat /etc/es-console/default.conf.tmpl | envsubst > /etc/nginx/conf.d/default.conf
```
Comment (marcoderama) [inline enterprise-suite/templates/es-console.yaml]: Just curious.  How is the environment initially setup so that `envsubst` has something to work with?

**Comment by jsravn** [comment]
Going to close this since its redundant with #114

**Pull request closed without merging**